airflow/operators/sensors.py

```diff
@@ -222,6 +222,60 @@ def poke(self, context):
         return count
 
 
+class NamedHivePartitionSensor(BaseSensorOperator):
+    """
+    Waits for a partition to show up in Hive. Only accepts plain partition
+    names (eg dt=xxx) for the partition argument and no filters.
```

Comment:
Only accepts fully specified partition identifier strings of the form `<DB name>.<table name>/<partition name>` e.g. `default.users/ds=2016-01-01`.